Tests/BugsnagErrorReportSinkTests.m

```diff
@@ -356,6 +357,42 @@ - (void)testHandledSerialization {
     XCTAssertNil(severityReason[@"attributes"]);
 }
 
+- (void)testHandledOverriddenSerialization {
+    BugsnagHandledState *state =
+    [BugsnagHandledState handledStateWithSeverityReason:HandledException];
+    state.unhandled = YES;
+    state.unhandledOverridden = YES;
+    NSDictionary *payload = [self reportFromHandledState:state];
+    
+    XCTAssertEqualObjects(@"warning", payload[@"severity"]);
```

Comment:
The arguments are the wrong way round; the expected value should be the second argument to `XCTAssertEqualObjects`

The same comment applies to the other uses of `XCTAssertEqualObjects` in this test method.